src/components/NicetyRow.js

```diff
@@ -9,7 +9,7 @@ const NicetyRow = React.createClass({
             <Row>
               {this.props.data
                   .map(function(result) {
-                      return (<Col lg="3" md="4" sm="6" xs="12">
+                      return (<Col lg="3" md="6" sm="6" xs="12">
```

Comment:
TIL about [React-Bootstrap's `<Col>` component](https://react-bootstrap.github.io/layout/grid/#auto-layout)! Neat.

It might still be worthwhile to look into flexbox in the hope of refactoring away `<NicetyRow>` and its associated grouping code, but that's way less important than most of the other improvements we could make.